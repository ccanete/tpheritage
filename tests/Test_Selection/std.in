S test 10 2 4 5
EXIT