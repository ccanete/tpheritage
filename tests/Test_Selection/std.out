OK
# Selection test of coordinates (10, 2) and (4, 5) has been created.

# The program is being closed.